optional-kubernetes-engine/bookshelf-frontend.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
 # The bookshelf frontend replica set ensures that at least 3
 # instances of the bookshelf app are running on the cluster.
 # For more info about Pods see:
-#   https://cloud.google.com/container-engine/docs/pods/
+#   https://cloud.google.com/kubernetes-engine/docs/pods/
```

Comment:
Are you updating this page too? Syncing all this sounds hard, ugh =(

Reply:
It'll be broken for a day or two. Then it'll be fixed. I'll be setting up redirects, too.